test/backend/routes.spec.js

```diff
@@ -269,9 +271,10 @@ describe('Authentication - ', () => {
             GET('settings')
             .then(res => {
                 assert.equal(res.status, 200);
-                done();
-            }).catch(done);
-        });
+                return;
+            });
+            done();
```

Comment:
This `done()` is outside of the `.then` that includes the `assert.equal(res.status, 200)` and so the `assert.equal` won't get called because the `done` will get called before it. So, should keep the `done()` in the `.then`